STYLE_GUIDE.adoc

```diff
@@ -0,0 +1,107 @@
+= Jakarta EE Tutorial Style Guide
+
+This document describes the requirements for new writing contributions to the Jakarta EE Tutorial;
+it includes a mixture of writing and https://docs.asciidoctor.org/asciidoc/latest/[Asciidoc] / https://docs.antora.org/antora/latest/[Antora] guidelines.
+
+== Eclipse Writing Style Guide
+
+New contributions must follow the https://www.eclipse.org/org/documents/writing-style-guide/[Eclipse Writing Style Guide] in addition to the guidelines mentioned herein.
+If there is a conflict, the guidelines in this document take precedence.
+
+== Tone and Voice
+
+Use a first-person voice but a professional tone and avoid slang.
+New tutorial material should be easy to read and approachable.
+
+For more guidelines, see https://www.eclipse.org/org/documents/writing-style-guide/#3[Tone and Voice].
+
+== Common Words, Names, and Abbreviations
+
+When using the following words, names, and abbreviations, consistently use the spelling and capitalization below:
+
+* "Basic authentication"
+* "Form authentication"
+* WAR
+* JAR
+* HTML
+* Profile names should be capitalized
+(Core Profile, Web Profile, etc.)
+
+== Using Hyphens
+
+Be aware of when to use a hyphen, especially for multiple-word adjectives before or after a noun. For example, "water-based paint" and "paint that is water based". See https://www.grammarly.com/blog/hyphen/[this article] and https://www.eclipse.org/org/documents/writing-style-guide/#21[Compound Words and Hyphenation] for details.
+
+== Lists
+
+For lists, use:
+
+* A colon before the list
+* Auto-numbering Asciidoc syntax
+* Avoid repeating the same word at the beginning of each bullet point
+(for example, "Use" or "Execute")
+* Use a period at the end of complete sentences, but don't use a period otherwise.
+
+For example:
+
+[source, asciidoc]
+----
+You'll learn how to:
+
+. Define security constraints
+. Provided authentication mechanisms
+. How to set a provided identity store that uses a database
+----
+
+See https://www.eclipse.org/org/documents/writing-style-guide/#32[Lists] for more details.
+
+== Internal Links
+
+You can easily link to other headings in the current document.
+For example, given the following heading:
+
+[source,asciidoc]
+----
+== This is My Header
+----
+
+You can link to it like so:
+
+[source,asciidoc]
+----
+Ths is a link to my header: <<_this_is_my_header>>
```

Comment:
Hmm would that still work when you edit "This is My Header" later to "This is My New Header"?

Reply:
Good point... I added a note explaining how to add an anchor... which tbh we should be doing, but isn't currently done anywhere.